front_end/content.py

```diff
@@ -907,6 +1008,12 @@ def has_duplicate_title(self, entries, this_entry, proposed_title):
                 return True
         return False
 
+    def save_course_passcode(self, course_id, passcode):
```

Comment:
The passcode should just be saved along with the other course details when we add/edit a course, rather than in its own function.